Apps/Sandcastle/gallery/3D Tiles Interactivity.html

```diff
@@ -48,7 +48,12 @@
     terrainProvider: Cesium.createWorldTerrain()
 });
 
-viewer.scene.globe.depthTestAgainstTerrain = true;
+var scene = viewer.scene;
+if(!scene.pickPositionSupported) {
```

Comment:
Style: we typically put a space between `if` and `(`.

`if (!scene.pickPositionSupported) {`